apps/edxapp/vars/vault/main.yml.j2

```diff
@@ -35,3 +35,7 @@ SECRETS_FILE:
   MONGODB_USER: "{{ MONGODB_USER }}"
   MONGODB_PASSWORD: "{{ MONGODB_PASSWORD }}"
 {% endraw %}
+
+# Uncomment the following key and add the content of your ssh private key if you
+# need to git-clone a private theme with via SSH.
+# EDXAPP_THEME_GIT_PRIVATE_KEY:
```

Comment:
I think it would help also to show how this key must be passed because it took us a few shots before doing it right. Something like this in the comment:
```
# If you need to git-clone a private theme with via SSH, uncomment the following key and add the
# content of your ssh private key in the following format:
#    + EDXAPP_THEME_GIT_PRIVATE_KEY: |
#    +   -----BEGIN RSA PRIVATE KEY-----
#    +   MIIJKAIBAAKCAgEAxY2VSZZRsjde+Hdpx1ixXhJV0MVCWkhTI76mMq9TRnXuihcY
#    +                               .../...
#    +   PpOyoKHbpoVxFDv4S9kj2RmmFDkWdTOncKNcVj4+ZkPGIcI8Hfcb04GpBW1PFfNn
#    +   -----END RSA PRIVATE KEY-----
```

Reply:
Totally true. Fixed.